sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlan.scala

```diff
@@ -65,6 +65,10 @@ abstract class SparkPlan extends QueryPlan[SparkPlan] with Logging with Serializ
     false
   }
 
+  override def verboseStringWithSuffix: String = {
+    s"$verboseString $outputPartitioning"
+  }
```

Comment:
This doesn't change anything that is in common use, one has to do `plan.treeString(verbose = true, addSuffix = true)` to get it. I would argue for keeping it for any future debugging.